spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,126 @@
+
+describe 'User Stories' do
+
+  let(:takeaway) { Takeaway.new }
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+  let(:checkout) { Checkout.new }
+  let(:item) { Item.new('Satay', 4.50) }
+  let(:item2) { Item.new('Spring rolls', 3.00) }
+  let(:quantity) { 3 }
+  let(:food) { 'Satay' }
+  let(:random_food) { Item.new('Turkish Delight', 4.00) }
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+describe 'Menu' do
+  describe '#get_item' do
+    it 'can look up an item' do
+      expect(menu.get_item(food).name).to eq food
+    end
+
+    it 'raises an error if the item is not on the menu' do
+      expect{ menu.get_item(random_food.name) }.to raise_error 'Sorry we don\'t have that on our menu'
```

Comment:
Metrics/LineLength: Line is too long. [102/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)